spec/rails_helper.rb

```diff
@@ -36,6 +36,7 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.include Capybara::DSL
```

Comment:
check if you guys have this - i needed to add it to run some specs in capybara but who knows 🤷‍♀